spec/helper_spec.rb

```diff
@@ -183,8 +183,7 @@
       ab_test('link_color', {'blue' => 0.01}, 'red' => 0.2)
       experiment = Split::ExperimentCatalog.find('link_color')
       expect(experiment.alternatives.map(&:name)).to eq(['blue', 'red'])
-      # TODO: persist alternative weights
-      # expect(experiment.alternatives.collect{|a| a.weight}).to eq([0.01, 0.2])
+      expect(experiment.alternatives.collect{|a| a.weight}).to eq([0.01, 0.2])
```

Comment:
Can you change `eq` for `match_array` or `contain_exatcly`?  See [here](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers)

It's might not always be true that the order is returned the same